### PR TITLE
Added minor fixes for exec_ceph_cmd function

### PR DESCRIPTION
**Pull request by keesturam**
- Added additional argument format for 'exec_ceph_cmd' function
 - Added ability to handle non munchable objects returned from ceph commands

### CODE REVIEW

File: ocs/ocp.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def exec_oc_cmd(self, command):
                 without the initial 'oc' at the beginning
 
         Returns:
-            Munch Obj: this object represents a returned yaml file
+            Munch Obj if the object represents a returned yaml file, else returns a list
```
Comment (RazTamir) [inline ocs/ocp.py]: When it will return a list?

Comment (keesturam) [reply ocs/ocp.py]: The output from Ceph commands is either in the form of dictionary or list. I've seen this in a couple of occasions.

The following is one such case.
```
sh-4.2# rbd ls rbd --format json 
["pvc-29e8c2cf-85d4-11e9-a278-0a334edd4bdc","pvc-771654f7-85d3-11e9-9454-02a058fe7dec"]
```
converting such an output to dict fails.
```
>   return [item.toDict() for item in out if item]
E   AttributeError: 'str' object has no attribute 'toDict'
```

File: ocs/ocp.py [hunk 2]
```diff
@@ -264,22 +267,24 @@ def get_ceph_tools_pod():
...
+    ceph_cmd += f" --format {format_arg}"
     out = ocp_pod_obj.exec_cmd_on_pod(ct_pod, ceph_cmd)
 
     # For some commands, like "ceph fs ls", the returned output is a list
-    if isinstance(out, list):
-        return [item.toDict() for item in out if item]
-    return out.toDict()
+    if isinstance(out, Munch):
+        return out.toDict()
+    return yaml.safe_load(out)
```
Comment (jilju) [inline ocs/ocp.py]:
```
return yaml.safe_load(out)
```
Is yaml.safe_load() still required? Here, 'out' will be a list

Comment (keesturam) [reply ocs/ocp.py]: out is a string here.

File: ocs/ocp.py (hunk 1 above)
Comment (clacroix12) [reply ocs/ocp.py]: Would it be better to always return a list than to have two different return types? Even if the list has one item, I think the consistency would be better than having any code that consumes this need to consider both cases.

Comment (RazTamir) [reply ocs/ocp.py]: I probably missing something.
This method is executing oc command so how 
```python
sh-4.2# rbd ls rbd --format json
```
is related to it?

**Comment by vasukulkarni** [comment]
@keesturam can you address the comments here and also due to new changes, it needs rebased or done as new PR

**Comment by vasukulkarni** [comment]
@keesturam please rebase and test.

**Comment by vasukulkarni** [comment]
@keesturam ping?

**Comment by keesturam** [comment]
@vasukulkarni sure. I should have this done in a couple of days.

**Comment by RazTamir** [comment]
@keesturam still relevant?
This PR is really old

**Comment by clacroix12** [comment]
Closing due to inactivity. Feel free to re-open / create a new PR to address these changes if necessary.

**Pull request closed without merging**